### PR TITLE
Add multiFilter support to front end

### LINKED ISSUE #260: Subsetting: Implement multi-filter support
We should be able to reuse the multifilter component from wdk-client.

### PR DESCRIPTION
**Pull request by dmfalke**
### Summary
This PR adds multiFilter support to the EDA Workspace. MultiFilters are a feature from the currently production ClinEpiDB website.

image

### Notes
* The filter chip has been simplified, as compared to the production site. This will need to be discussed in a UX meeting.
* The backend API could be updated to handle requesting distributions for multiple variables in one go.
* ~~We unnecessarily refresh counts when changes are made to the active multiFilter. This will be addressed shortly.~~

---

~~depends on https://github.com/VEuPathDB/WDKClient/pull/148~~
fixes #260

### CODE REVIEW

**Comment by dmfalke** [comment]
I almost forgot about a necessary update to [wdk-client](https://github.com/VEuPathDB/WDKClient). I will update soon.

**Comment by dmfalke** [comment]
Per the UX meeting today:

- [ ] Update counts for sub-variables based on selection. Take operation into account.
- [ ] Add "Update" button for the above.

**Comment by dmfalke** [comment]
Spoke w/ Steve. Will emulate the live site, which is to update sibling counts only if operation is **all**. The case for **any** is more challenging. I will open a new issue for this, with details.

**Review by bobular — APPROVED** [review]
Looks good and works well for me.

Was it intentional to not have the "22,566 (>99%) of 22,567 Household have data for this variable"  above each variable like in the live clinepi site?

File: src/lib/core/components/filter/FilterContainer.tsx [hunk 1]
```diff
@@ -25,17 +26,20 @@ export function FilterContainer(props: Props) {
     <HistogramFilter {...props} />
   ) : narrowProps(isTableVariable, props) ? (
     <TableFilter {...props} />
+  ) : narrowProps(MultiFilterVariable.is, props) ? (
```
Comment (bobular) [inline src/lib/core/components/filter/FilterContainer.tsx:29]: the `IOTSType.is` seems cool :+1:

**Review by bobular — APPROVED** [review]
> Looks good and works well for me.
> 
> Was it intentional to not have the "22,566 (>99%) of 22,567 Household have data for this variable" above each variable like in the live clinepi site?

Ah from MultiFieldFilter.jsx - it looks like if there is 100% data, this is skipped.

```
      : unknownCount > 0 && (
        <div style={{ fontWeight: 300 }}>
          <b>{toPercentage(row.summary.internalsCount,this.props.dataCount).toLocaleString()}% of {this.props.dataCount.toLocaleString()}</b> {this.props.displayName} have data  
        </div>
```

File: src/lib/core/components/filter/FilterContainer.tsx (hunk 1 above)
Comment (dmfalke) [reply src/lib/core/components/filter/FilterContainer.tsx:29]: :smile:

**Comment by dmfalke** [comment]
This should be good to go now. Once I merge the latest from `main`, I will likely merge this PR.

**Pull request merged**